modules/common/src/DatasetInfo.php

```diff
@@ -220,6 +220,7 @@ protected function getResourcesInfo(\stdClass $distribution) : array {
       'fetcher_percent_done' => $info->fileFetcherPercentDone ?? 0,
       'file_path' => isset($fileMapper) ? $fileMapper->getFilePath() : 'not found',
       'importer_status' => $info->importerStatus,
+      'importer_status' => $info->importerError,
```

Comment:
This will need to be changed since error is overwriting status.
```suggestion
      'importer_status' => $info->importerError ?: $info->importerStatus,
```